src/ServerTrait.php

```diff
@@ -35,7 +35,7 @@ private function stopWebServer(): void
     private function pause($message): void
     {
         if (\in_array('--debug', $_SERVER['argv'], true)
-            && $_SERVER['PANTHER_NO_HEADLESS'] ?? false
+            && ($_SERVER['PANTHER_NO_HEADLESS'] ?? false)
```

Comment:
Violations of this type were actually a bug - `true && $var ?? false` is equivalent to `(true && $var) ?? false` instead of `true && ($var ?? false)`.